src/components/DayPicker.jsx

```diff
@@ -434,6 +443,14 @@ class DayPicker extends BaseClass {
         }
         break;
 
+      case 'Tab':
+        if (e.shiftKey) {
+          onShiftTab();
+        } else {
+          onTab();
```

Comment:
i wonder if maybe we should have one `onTab` function, that passes a list of the modifier keys pressed?